BFWControls/Modules/Carousel/Controller/CarouselViewController.swift

```diff
@@ -129,14 +129,16 @@ open class CarouselViewController: UICollectionViewController {
         scroll(toPage: pageControl.currentPage, animated: true)
     }
     
-    fileprivate func updatePageControl() {
+    fileprivate func updatePageControl(_ shouldUpdateCurrentPage: Bool = true) {
```

Comment:
Please remove the `_` so that the parameter is labelled.

eg calling `updatePageControl(true)` gives no indication as to what is `true`.